api/src/main/java/org/msf/module/visit/CloseVisitOnAnOutcomeTask.java

```diff
@@ -1,20 +1,24 @@
 package org.msf.module.visit;
 
 import org.bahmni.module.bahmnicore.service.BahmniObsService;
-import org.openmrs.Concept;
-import org.openmrs.Visit;
+import org.openmrs.*;
```

Comment:
@binduak @vinishad It is recommended to use specificType instead of * while importing, to avoid ambiguity. 
Ref: https://stackoverflow.com/questions/187453/import-package-vs-import-package-specifictype

Reply:
Sure